docs/markdown/Creating-releases.md

```diff
@@ -58,3 +58,20 @@ for example when done in CI that already does its own testing.
 So with `--no-tests` you can tell Meson "Do not build and test generated
 packages.".
 
+## Release a subproject separately
+
+*Since 0.57.0* the `meson dist` command can now create a distribution tarball
+for a subproject in the same git repository as the main project. This can be
+useful if parts of the project (e.g. libraries) can be built and distributed
+separately. In that case they can be moved into `subprojects/mysub` and running
+`meson dist` in that directory will now create a tarball containing only the
+source code from that subdir and not the rest of the main project or other
+subprojects.
+
+For example:
+```sh
+git clone https://github.com/myproject
+cd myproject/subprojects/mysubproject
+meson builddir
+meson dist -C builddir
+```
```

Comment:
This should also mention what the outcome will be. Something like "this will create foo.tar.gz and/or baz.tar.gz in directories xyz".

This should also be rebased so CI is rerun. There's quite a lot of red currently.

Reply:
Done.

Reply:
All the red is still there ... 😞